package.nls.json

```diff
@@ -0,0 +1,96 @@
+{
+    "cmake-tools.command.cmake.viewLog.title": "Open the CMake Tools log file",
+    "cmake-tools.command.cmake.editKits.title": "Edit user-local CMake kits",
+    "cmake-tools.command.cmake.scanForKits.title": "Scan for Kits",
+    "cmake-tools.command.cmake.selectKit.title": "Select a Kit",
+    "cmake-tools.command.cmake.setVariant.title": "Set the current build variant",
```

Comment:
Issue #743 requested that this command name be shortened.  Can you do this now?  Either "Set Variant" or "Select Variant" (based on "Select a Kit") seem ok.